curta/src/chip/hash/sha/sha256/mod.rs

```diff
@@ -103,6 +110,7 @@ impl<L: AirParameters> AirBuilder<L> {
 
         // Get the w value from the bus
         let w_challenges = self.alloc_challenge_array::<CubicRegister>(U32Register::size_of() + 1);
+        // no value in w_window yet so what this w_window.get(0) does?
```

Comment:
it gets a "pointer" to the value that could live in w_window

Reply:
yeah, understand it now.